recovery_reader.go

```diff
@@ -28,17 +28,17 @@ func (fr *fileRecovery) readEntry() (e *Entry, err error) {
 	if err != nil {
 		return nil, err
 	}
-	meta := readMetaData(buf)
 
 	e = &Entry{
-		crc:  binary.LittleEndian.Uint32(buf[0:4]),
-		Meta: meta,
+		crc: binary.LittleEndian.Uint32(buf[0:4]),
 	}
+	e.ParseMeta(buf)
```

Comment:
[golangci] reported by reviewdog 🐶
Error return value of `e.ParseMeta` is not checked (errcheck)

Raw Output:
recovery_reader.go:35:13: Error return value of `e.ParseMeta` is not checked (errcheck)
	e.ParseMeta(buf)
	           ^